drivers/clk/zynqmp/pll.c

```diff
@@ -279,11 +279,19 @@ static int zynqmp_pll_enable(struct clk_hw *hw)
  */
 static void zynqmp_pll_disable(struct clk_hw *hw)
 {
+        u32 reg;
```

Comment:
incorrect indentation here, should be tabs: https://www.kernel.org/doc/html/v4.10/process/coding-style.html